code/obj/effect.dm

```diff
@@ -12,5 +12,12 @@
 		..()
 		src.render_target = "*\ref[src]"
 
+/obj/effect/rt
```

Comment:
What's the benefit of this over `/obj/effect/distort`? It seems to serve the same purpose but is named more confusingly and is lacking crucial appearance_flags that should be there.

Reply:
Didn't know why the flags weren't in /obj/effect, and the masks worked without the flags, so they never got put in.
Renaming /distort sounds fine though, it looks like its default icon isn't actually used anywhere.